tests/factory_fixtures/tenant.py

```diff
@@ -1,13 +1,15 @@
 import pytest
+from faker import Faker
 from models.tenant import TenantModel
 
 @pytest.fixture
 def create_tenant(create_property):
     def _create_tenant(property=create_property()):
+        fake = Faker()
         tenant = TenantModel(
-                firstName="firstName",
-                lastName="lastName",
-                phone="1234567890",
+                firstName=fake.unique.first_name(),
+                lastName=fake.unique.last_name(),
```

Comment:
There are not unique constraints for the firstName and lastName on the tenant model. People are allowed to have the same name. so I don't think using unique here is necessary.

Also, the way Faker is being used here will not guarantee unique data. Pytest has built-in support for Faker and it is available as factory. We should be using the faker factory so that when we do call unique we can be sure we will get unique values. Plus, the faker factory should run faster than having to instantiate a Faker object each time we call a factory.